README.md

```diff
@@ -184,6 +184,7 @@ Currently **officially** using Airflow:
 1. [JobTeaser](https://www.jobteaser.com) [[@stefani75](https://github.com/stefani75) &  [@knil-sama](https://github.com/knil-sama)]
 1. [Kalibrr](https://www.kalibrr.com/) [[@charlesverdad](https://github.com/charlesverdad)]
 1. [Karmic](https://karmiclabs.com) [[@hyw](https://github.com/hyw)]
+1. [King.com](https://www.king.com)
```

Comment:
could you follow the other format(e.g [Karmic](https://karmiclabs.com) [[@hyw](https://github.com/hyw)])?